### PR TITLE
Jest testing

### PR DESCRIPTION
**Pull request by LuisDavidRodriguez**
## Add jest, Testing add and delete functions
- We install and set up the Jest framework
- Create 2 new directories __tests__ __mocks__
- Create a activity.test.js file to write inside the tests
- Test the functions add and delete
- Providing the source code for the test into the mocks directory
- Using the jest-environment **jsdom**

### Hi there dear reviewer
giphy (4)

### CODE REVIEW

File: src/__tests__/activities.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * @jest-environment jsdom
+ */
+/* eslint-disable*/
+import ActivitiesManager from '../__mocks__/activities.js';
+
+describe('Task1', () => {
+
+  it('Testing activities Manager Add', () => {
+    document.body.innerHTML = 
+    '<div>' +
```
Comment (mikethreels) [inline src/__tests__/activities.test.js]: - [x] in this case, you are only testing the mock functions you wrote instead of the actual code.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__tests__/activities.test.js (hunk 1 above)
Comment (LuisDavidRodriguez) [reply src/__tests__/activities.test.js]: Ok yeah that is true so if we just import from the normal file that should be okay right? to test not the mocks but the real one @mikethreels

**Comment by LuisDavidRodriguez** [comment]
## Hi there dear reviewer
- we just change the path and delete the mocks directory because at the end it was not necessary, hence we are now testing the original file

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @LuisDavidRodriguez, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**